testdata/project-v4-with-plugins/dist/helm-install.yaml

```diff
@@ -0,0 +1,1161 @@
+---
```

Comment:
I understand that you added this to make it easier to review the changes, and it seems like an interesting approach.

However, it appears this was done manually. All the files generated under testdata are created using the shell script found at [generate.sh](https://github.com/kubernetes-sigs/kubebuilder/blob/master/test/testdata/generate.sh). We cannot manually add or modify content in this directory. Instead, the logic for generating these files needs to be incorporated into the script. Also, the action that calls it will need to have helm installed. We check if the testdata is updated with the PR changes to allow get it merged see: https://github.com/kubernetes-sigs/kubebuilder/blob/master/.github/workflows/testdata.yml

Additionally, if we decide to include this change, we must ensure that the helm-install.yaml file is re-generated when running the [Makefile target](https://github.com/kubernetes-sigs/kubebuilder/blob/master/Makefile#L91-L94). This ensures consistency and alignment with the existing build and test automation processes.

Reply:
I think this change would need to be in an PR only for that.

Reply:
Hi @camilamacedo86,
I've used `make generate` to update the files in the testdata folder, it was not done manually. Those files are being done based on the changes I've made to the .go files.